### DISCUSSION TITLE
World Chat

### DISCUSSION THREAD

**Discussion by Selyria** [azerothcore/azerothcore-wotlk#3972] [General]
Hi I wanted to ask if there is a module with which you can open a world-chat 

Something like this.

https://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-emulator-servers/wow-emu-general-releases/780187-release-advanced-world-chat-system-3-3-5a-wotlk.html#post4017816

I'd be happy to answer that.

**Comment by Yehonal** [comment]
Currently not, but since it's a simple script you can easily create it by your self and share with AzerothCore organization that will help you to improve it.

Here's the how to create a module: http://www.azerothcore.org/wiki/Create-a-Module

**Comment by BarbzYHOOL** [comment]
Isn't there one by Kraft though?

**Comment by wizzymore** [comment]
Well, actually that is my module and i am happy to let you know that it is imported to AzerothCore!
https://github.com/wizzymore/mod_world_chat

**Comment by Helias** [comment]
Wonderful, submit it in the azerothcore modules-catalogue :wink: 
http://www.azerothcore.org/modules-catalogue/submit.html

**Comment by BarbzYHOOL** [comment]
Awesome @wizzymore  I forked it because I am used to do this with proper modules but that's gonna be a duplicate in the catalogue no?
